packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -187,9 +224,21 @@ test('PUT replaces an existing collection', async (t) => {
     created_at: originalPgRecord.created_at,
     updated_at: actualPgCollection.updated_at,
   });
+
+  const { Messages } = await sqs().receiveMessage({
+    QueueUrl: t.context.QueueUrl,
+    WaitTimeSeconds: 10,
+  }).promise();
+
+  t.is(Messages.length, 1);
+
+  const message = JSON.parse(JSON.parse(Messages[0].Body).Message);
+  const omitFields = ['createdAt', 'updatedAt'];
+  t.is(message.event, 'Update');
+  t.deepEqual(omit(message.record, omitFields), omit(actualCollection, omitFields));
```

Comment:
There is no need to `omit` if we're comparing against the fetched record from dynamo which has the timestamps

Reply:
[`bdd6661` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/bdd66615c308abb672a3d4ca00c6eb92cb5eef48#diff-b87875b83c4639dd8008bd26f239a6580ba66ed6ab9441aac8c86575d8b7ab73R236)